.github/workflows/benchmarks.yml

```diff
@@ -92,7 +92,9 @@ jobs:
         with:
           r-version: ${{ matrix.r-version }}
       - name: "Set up LuaJit"
-        uses: leafo/gh-actions-lua@v8.0.0
+        run: |
+          cd ~/work/Microbenchmarks/Microbenchmakrs/lua
+          ./lua-install.sh
         with:
           luaVersion: luajit-${{ matrix.lua-version }}
```

Comment:
^Should be deleted if you are skipping the gh-actions-lua workflow.

Reply:
See previous comments